src/social-web/routes/feed/inspector.tsx

```diff
@@ -112,6 +120,24 @@ export default function FeedInspector( { id, onClose }: FeedInspectorProps ) {
 					{ ( post.content?.rendered || post.excerpt?.rendered ) && (
 						<RenderHTML html={ post.content?.rendered || post.excerpt?.rendered || '' } />
 					) }
+					{ tags && tags.length > 0 && onTagClick && (
+						<div className="activitypub-inspector-tags">
+							{ tags.map( ( tag: any ) => (
+								<Button
+									key={ tag.id }
+									variant="secondary"
+									size="compact"
+									onClick={ () => {
+										onTagClick( tag.id );
+										onClose();
+									} }
+									className="activitypub-inspector-tag"
+								>
+									#{ tag.name }
+								</Button>
+							) ) }
+						</div>
+					) }
```

Comment:
This will duplicate tags to the content of the post. We should probably fix that in the way we parse and save ap_posts.

<img width="293" height="268" alt="Image" src="https://github.com/user-attachments/assets/9a2ab9cf-cced-429e-918d-a063158ebd7f" />